Renderer.js

```diff
@@ -0,0 +1,62 @@
+class Renderer {
+  loadInfo(data) {
+    const source = $("#details-template").html();
+    const template = Handlebars.compile(source);
+
+    // const pName = template({name:data.name})///// i kept these comments in order to discuss the syntax with you guys!
+    // const pAddress = template({address:data.address})
+    // const pPicture = template({picture:data.picture})
+
+    const newHtml = template({
+      name: data.name,
+      address: data.address,
+      picture: data.picture,
+    });
+    $(".user-container").empty().append(newHtml);
+    // $(".user-container").append(pAddress)
+    // $(".user-container").append(pPicture)
+  }
+
+  loadFriends(friends) {
+    const source = $("#friends-template").html();
+    const template = Handlebars.compile(source);
+    $(".friends-container").empty();
+    for (let friend of friends) {
```

Comment:
you do not need to loop over the friends array. you can send the array to handlebars and use the {{# each friends}} in the HTML template